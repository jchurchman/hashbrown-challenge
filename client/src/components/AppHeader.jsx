import React from 'react'
import styled from 'styled-components'

const Header = styled.div`
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  display: flex;
  justify-content: center;
  background-color: #ECEBEB;
`

const HeaderBar = styled.header`
  position: absolute;
  top: 0;
  left: 0;
  right: 0;
  height: 4em;
  background-color: teal;
  display: flex;
  align-items: center;
  justify-content: center;
`

const HeadingText = styled.h1`
  color: white;
  margin: 0;
`

const StrikeText = styled.span`
  text-decoration: line-through;
  margin: 0 0.35em;
`

const HashbrownTag = styled.img`
  width: auto;
  height: 8em;
  border-radius: 3em;
  margin-top: 5em;
`

const AppHeader = () => {
  return (
    <Header>
      <HeaderBar>
        <HeadingText>
          Get your
          <StrikeText>
            Hashbrowns
          </StrikeText>
          Hashtags here!
        </HeadingText>
      </HeaderBar>
      <HashbrownTag src={"https://media.giphy.com/media/10GUbOX16lS15C/giphy.gif"} alt="hashbrown" />
    </ Header>
  )
}

export default AppHeader