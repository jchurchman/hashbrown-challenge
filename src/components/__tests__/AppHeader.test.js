import renderer from 'react-test-renderer';
import AppHeader from "../AppHeader";

describe('AppHeader ', () => {
  it('renders as expected ', () => {
    const tree = renderer
      .create(<AppHeader />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})