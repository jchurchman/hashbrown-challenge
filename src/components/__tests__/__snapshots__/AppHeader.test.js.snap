// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader  renders as expected  1`] = `
<div
  className="sc-bdnxRM cXXUKy"
>
  <header
    className="sc-gtsrHT fYUfFJ"
  >
    <h1
      className="sc-dlnjwi jeaIPE"
    >
      Get your
      <span
        className="sc-hKFxyN jZshyA"
      >
        Hashbrowns
      </span>
      Hashtags here!
    </h1>
  </header>
  <img
    alt="hashbrown"
    className="sc-eCApnc Hxeox"
    src="https://media.giphy.com/media/10GUbOX16lS15C/giphy.gif"
  />
</div>
`;
